src/less/cta-button/cta-button.less

```diff
@@ -58,22 +58,32 @@ span.cta-btn__cell--fixed-height svg.icon {
     width: 1rem;
 }
 
+a.cta-btn--truncated {
+    .btn-truncate();
+}
+
 a.cta-btn--large {
-    border-radius: calc(@button-height-large / 2);
+    border-radius: @button-border-radius-large;
+    display: inline-flex;
```

Comment:
Hmm...what's this for? Is it to account for icon positioning inside?

Reply:
Yes, that is correct. `display: inline-flex;` has been applied to `a.cta-btn--large`, `a.cta-btn--large-truncated`, `a.cta-btn--large-fixed-height`